PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -161,7 +162,7 @@ def miniAOD_customizeCommon(process):
     )
 
     addJetCollection(process, postfix   = "", labelName = 'Puppi', jetSource = cms.InputTag('ak4PFJetsPuppi'),
-                    jetCorrections = ('AK4PF', ['L1FastJet', 'L2Relative', 'L3Absolute'], ''),
+                    jetCorrections = ('AK4PFchs', ['L2Relative', 'L3Absolute'], ''),
                     algo= 'AK', rParam = 0.4, btagDiscriminators = map(lambda x: x.value() ,process.patJets.discriminatorSources)
                     )
     
```

Comment:
@gpetruc, as @ferencek mentioned, SV TagInfos are not saved in slimmedJets and slimmedJetsPuppi (only slimmedJetsAK8), so we don't need to save them in patJets. Similarly, L120-L121 can be removed for patJets too.
I can do a check on the disk space usage of this branch.